Jessy/stocks_challenge.py

```diff
@@ -0,0 +1,113 @@
+stock = input()
+print()
+print(stock+",is a great choice")
+
+print("Challenge 3.2.2: Perform user-specific calculations")
+# TODO: You have all 3 user inputs stores in variables. Based on that, write conditional (if-elif-else) statements to find out the number of stocks of the company that can be purchased with the savings amount.
+# number_of_stocks
+# number_of_stocks== (int(savings)/ amazon)
+# number_of_stocks=1
+# print(number_of_stocks==(int(savings)/amazon))
+if stock == "amzn":
+    print("excellent choice")
+    number_of_stocks = (int(savings) / amazon)
+    if number_of_stocks >1:
+        print(f"{name} has $ {savings} therefore he can purchase {int(number_of_stocks)} of Amazon")
+    elif number_of_stocks<1:
+        print("Sorry, you dont have enough savings")
+elif stock == "aapl":
+    print("excellent choice")
+    number_of_stocks = (int(savings) / apple)
+    if number_of_stocks >1:
+        print(f"{name} has $ {savings} therefore he can purchase {int(number_of_stocks)} of Apple")
+    elif number_of_stocks<1:
+        print("Sorry, you dont have enough savings")
+elif stock == "fb":
+    print("excellent choice")
+    number_of_stocks = (int(savings) / fb)
+    if number_of_stocks >1:
+        print(f"{name} has $ {savings} therefore he can purchase {int(number_of_stocks)} of FaceBook")
+    elif number_of_stocks<1:
+        print("Sorry, you dont have enough savings")
+elif stock == "goog":
+    print("excellent choice")
+    number_of_stocks = (int(savings) / google)
+    if number_of_stocks >1:
+        print(f"{name} has $ {savings} therefore he can purchase {int(number_of_stocks)} of Google")
+    elif number_of_stocks<1:
+        print("Sorry, you dont have enough savings")
+elif stock == "msft":
+    print("excellent choice")
+    number_of_stocks = (int(savings) / msft)
+    if number_of_stocks >1:
+        print(f"{name} has $ {savings} therefore he can purchase {int(number_of_stocks)} of Microsoft")
+    elif number_of_stocks<1:
+        print("Sorry, you dont have enough savings")
+else:
+    print("Sorry, Those stocks are not available")
+
+# number_of_stocks= 0
+# if stock == "aapl":
+#      print("excellent choice")
+#     number_of_stocks = (savings / apple)
+# elif number_of_stocks >1:
+#     print(f"{name} has $ {savings} therefore you can purchase {number_of_stocks} of apple")
+# else 
+#     print(f"Sorry, don't have enough to buy stocks right now")
+
+
+'''
+Your code should look like this:
+
+if stock == "amzn":
+    ...
+elif ...
+else ...
+'''
+
+print()
+
+print("Challenge 3.2.3: Output for the user the result")
+# TODO: COnce you have calculated the number of stocks that can be purchased, print the result for the client. Result should be in a format like this:
+
+# Alex has $5000 in savings and he can buy 50 shares of Apple at the current price of $100.
```

Comment:
Nice if-blocks, how was running this code? Did it run the way you intended? I'm only able to view these for now